src/control-plane/backend/lambda/api/test/api/ddb-mock.ts

```diff
@@ -257,7 +257,10 @@ function dictionaryMock(ddbMock: any, name?: string): any {
             type: 'PLUGIN#BUILT-IN-1',
             prefix: 'PLUGIN',
             name: 'Transformer',
-            description: 'Description of Transformer',
+            description: {
```

Comment:
In future, we might read the dict.json for generating mock data.

Reply:
#465